feedparser/api.py

```diff
@@ -220,9 +222,11 @@ def parse(url_file_stream_or_string, etag=None, modified=None, agent=None, refer
     result['headers'].update(response_headers or {})
 
     data = convert_to_utf8(result['headers'], data, result)
+    use_json_parser = result['content-type'] == 'application/json'
```

Comment:
this won't work in case when the content type is `'application/feed+json'`. the line should be:
```python
use_json_parser = result['content-type'] in ('application/feed+json', 'application/json')
```
ideally we would reuse the `json_content_types` used inside `convert_to_utf8` function

Reply:
Good point, I'll incorporate this feedback!